### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rande**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,5 @@ License
 -------
 
 This bundle is available under the [MIT license](Resources/meta/LICENSE).
+
+this is a test
```
Comment (Soullivaneuh) [inline README.md]: This* and a dot.

:-P

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Comment by rande** [comment]
ok to test

**Pull request closed without merging**